platform/darwin/test/MGLCodingTests.m

```diff
@@ -80,6 +80,32 @@ - (void)testPolyline {
     XCTAssertEqual([multiLine coordinate].latitude, multiLineCenter.latitude);
     XCTAssertEqual([multiLine coordinate].longitude, multiLineCenter.longitude);
     
+    CLLocationCoordinate2D segmentCoordinates[] = {
+        CLLocationCoordinate2DMake(35.040390, -85.311477),
+        CLLocationCoordinate2DMake(35.040390, -85.209510),
+    };
+    
+    NSUInteger segmentCoordinatesCount = sizeof(segmentCoordinates) / sizeof(CLLocationCoordinate2D);
+    MGLPolyline *segmentLine = [MGLPolyline polylineWithCoordinates:segmentCoordinates count:segmentCoordinatesCount];
+    CLLocationCoordinate2D segmentCenter = CLLocationCoordinate2DMake(35.0404006631, -85.2604935);
+    
+    XCTAssertEqualWithAccuracy([segmentLine coordinate].latitude, segmentCenter.latitude, 1);
```

Comment:
The third parameter to this macro is a float with the same magnitude as the maximum delta, not the number of decimal places. So here we’re only testing that the center between 35.040390 and 35.040390 rounded to the ones place is 35. One degree of latitude or longitude is quite a big difference; we should assert with much higher accuracy.